code/modules/surgery/organs_internal.dm

```diff
@@ -453,13 +453,13 @@
 	name = "cut carapace"
 	allowed_tools = list(
 	/obj/item/weapon/scalpel = 100,		\
-	/obj/item/weapon/kitchen/knife = 75,	\
-	/obj/item/weapon/shard = 50, 		\
-	/obj/item/weapon/scissors = 10,		\
-	/obj/item/weapon/twohanded/chainsaw = 1, \
-	/obj/item/weapon/claymore = 5, \
-	/obj/item/weapon/melee/energy/ = 5, \
-	/obj/item/weapon/pen/edagger = 5, \
+	/obj/item/weapon/kitchen/knife = 90,	\
+	/obj/item/weapon/shard = 60, 		\
+	/obj/item/weapon/scissors = 12,		\
+	/obj/item/weapon/twohanded/chainsaw = 2, \
```

Comment:
👌

Reply:
Don't look at me, it's like that already! I just changed the numbers like the original PR.